### ISSUE TITLE
Widgets not showing in Jupyter lab

### ISSUE THREAD

**Issue by pinshuai** [pyvista/pyvista#332]
Hi, I am testing the examples for the IPython tools (https://docs.pyvista.org/plotting/ipy_tools.html). The plot showed up in a new window, but no widgets (i.e., slider, dropdown ...) were shown in the notebook.

Here is an example code I am trying to duplicate:

```py
import pyvista as pv
from pyvista import examples

dataset = examples.load_hexbeam()

# Use the slicer tool
pv.OrthogonalSlicer(dataset)
```

Did I miss anything?

**Comment by banesullivan** [comment]
That's very strange... could you share your system information by copy/pasting the output of this from that same Jupyter notebook:

```py
import pyvista as pv
print(pv.Report())
```

**Comment by pinshuai** [comment]
Here are the screen output.

```

--------------------------------------------------------------------------------
  Date: Mon Jul 29 21:13:31 2019 PDT

            Darwin : OS
                 8 : CPU(s)
            x86_64 : Machine
             64bit : Architecture
           Jupyter : Environment

  Python 3.7.3 (default, Mar 27 2019, 16:54:48)  [Clang 4.0.1
  (tags/RELEASE_401/final)]

            0.21.2 : pyvista
             8.1.2 : vtk
            1.16.4 : numpy
             2.5.0 : imageio
             1.4.3 : appdirs
             0.4.3 : scooby
             3.1.0 : matplotlib
            5.11.3 : PyQt5
             7.6.1 : IPython
             7.5.0 : ipywidgets
             0.6.0 : panel
--------------------------------------------------------------------------------
```

**Comment by banesullivan** [comment]
Thanks! And are you using Jupyter notebook or lab?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by pinshuai** [comment]
Jupyterlab

**Comment by banesullivan** [comment]
Ah - this is not good. I can confirm that none of the IPython tools work in Jupyter lab - I will have to fix this ASAP.

In the meantime, Jupyter notebook should work.

**Comment by banesullivan** [comment]
Update: installing the `ipywidgets` extension to Jupyter lab should fix the issue. [See installation instructions](https://ipywidgets.readthedocs.io/en/latest/user_install.html#installing-the-jupyterlab-extension)

I cannot confirm this as a solution as the install is hung on something for me (this happens to me everytime I try to install something to jupyter lab which is why I never use/test it)

**Comment by pinshuai** [comment]
I tried re-installing `ipywidgets` and `widgetsnbextension`, and also the labextension by

```
jupyter labextension install @jupyter-widgets/jupyterlab-manager
```

The widgets still did not show up.

**Comment by banesullivan** [comment]
I managed to get my jupyter lab rebuilt and I'm in the same boat - I cannot figure out why this isn't working...

![Screen Shot 2019-07-30 at 11 52 19 AM](https://user-images.githubusercontent.com/22067021/62152796-7e10b580-b2c0-11e9-9f05-ef5e19d2c3f5.png)


Maybe https://github.com/jupyter-widgets/ipywidgets/issues/2514#issuecomment-515226035 provides insight? I tried downgrading to `ipywidgets=7.4.2` but had the same issue.

-----

We're really doing nothing special with `ipywidgets` in PyVista. We're basically just running the following inside a module - which FYI, the following also doesn't work for me in Jupyter lab so I'm inclided to think this is a Jupyter lab/ipywidgets issue:

```py
from ipywidgets import interact, interactive
import ipywidgets as widgets

def f(x):
    print(x)

interact(f, x=widgets.FloatSlider(min=0, max=10))
```

![Screen Shot 2019-07-30 at 11 55 55 AM](https://user-images.githubusercontent.com/22067021/62153108-20c93400-b2c1-11e9-926c-56e4b3b99293.png)


But there is no issue in Jupyter **notebook**:

![Screen Shot 2019-07-30 at 11 57 39 AM](https://user-images.githubusercontent.com/22067021/62153198-46563d80-b2c1-11e9-94ea-e9d1e113d538.png)
